.github/workflows/update-dask-sql.yml

```diff
@@ -0,0 +1,53 @@
+# Updates stable dask-sql version to use in images.
+# Runs once daily.
+
+name: Check for new dask-sql version
+
+on:
+  schedule:
+    - cron: "0 0 * * *" # Daily “At 00:00” UTC
+  workflow_dispatch:
+
+jobs:
+  update-dask-sql:
+    runs-on: ubuntu-latest
+    if: github.repository == 'rapidsai/docker'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get old dask-sql version
+        id: old_version
+        uses: the-coding-turtle/ga-yaml-parser@v0.1.1
+        with:
+          file: settings.yaml
+          key: STABLE_DASK_SQL_VERSION
```

Comment:
Is it safe to say that the `STABLE_DASK_SQL_VERSION` in [settings.yaml](https://github.com/rapidsai/docker/blob/branch-22.04/settings.yaml) will always match up with the `DASK_SQL_VER` used in the axis YAMLs?

Reply:
yes!